docs/code-reviews/faq.md

```diff
@@ -4,18 +4,34 @@ This is a list of questions / frequently occurring issues when working with code
 
 ## What makes a code review different from a PR?
 
+A pull request is a way to notify a feature is finished and can be merged into the master branch.  
+A code review is having someone go over the code in the pull request and validate it before it is merged.
+
 | Code Review | Pull Request |
 --- | ---
 | Source code focused | Intended to enhance and enable code reviews. Includes both source code but can have a broader scope (e.g., docs, integration tests, compiles) |
 | Intended for **early feedback** before submitting a PR | Not intended for **early feedback**. Created when author is ready to merge |
 | Usually a synchronous review with faster feedback cycles (draft PRs as an exception). Examples: scheduled meetings, over-the-shoulder review, pair programming | Usually a tool assisted asynchronous review but can be elevated to a synchronous meeting when needed |
 
-## We experience very large PRs, how can we fix this?
+## Why do we need code reviews?
+
+Our peer code reviews are structured around best practices, to find specific kinds of errors. Much like you would still run a linter over mobbed code, you would still ask someone to make the last pass to make sure the code conforms to expected standards and avoids common pitfalls.
+
+## How to perform a review?
```

Comment:
I feel like we are losing a bit of information and context here. Perhaps we should revert these changes a little. I suggest moving this section to its old place (not including the one above ["Why do we need code reviews?"]) and combine the old content under one heading e.g., "We pair or mob. How should this reflect in our code reviews?"

Reply:
Fixed.